### ISSUE TITLE
The problem of dividing the training and test sets

### ISSUE THREAD

**Issue by wujiekd** [Vision-CAIR/artelingo#3]
When we used the data preprocessing file, we found that the training set did not separate the test set, which resulted in an evaluation that should be inaccurate.
